### ISSUE TITLE
Automate Confussion matrix, Roc Curves, Accuracy plots codes.

### ISSUE THREAD

**Issue by yvkrishna** [flow2ml/Flow2ML#50]
After training a model, we need to check it using few result metrics like Accuracy plots codes, confusion matrix, Roc Curves, etc. 

This issue focuses on coding the functions for those metrics: 
- For Accuracy plots: 
   - Take the trained model as input and save the model accuracy plots based on the metrics the user has chosen while training.
- For Confusion matrix and Roc Plots: 
   - Take the trained model and test data as inputs and fetch the predicted results. Compare those values to truth values and generate corresponding confusion matrix and roc curves.

**Comment by saniaarora15** [comment]
Can I work on this under Gssoc21 ?

**Comment by yvkrishna** [comment]
yes @saniaarora15 , you can work on this issue and update it in this thread

**Comment by chebroluharika** [comment]
@saniaarora15 do we have any update on this?

**Comment by saniaarora15** [comment]
It will be done by 1st may.

**Comment by chebroluharika** [comment]
@saniaarora15 are you on track?
